main.py

```diff
@@ -18,24 +19,25 @@ def signup():
     password = request.form["password"]
     verify = request.form["verify"]
     email = request.form["email"]
-    string_len = True
+    
+    username_error = ""
+    password_error = ""
+    verify_error = ""
+    email_error = ""
+    
     if username == "" or check_character_len(username) == False:
         username_error = "That's not a valid username"
-    else:
-        username_error = ""
+
     if password == "" or check_character_len(password) == False:
         password_error = "That's not a valid password"
-    else:
-        password_error = ""
+
     if verify == "" or verify != password or check_character_len(verify) == False:
         verify_error = "Passwords don't match"
```

Comment:
We shouldn't need to check to see if the verification password meets the length requirements, if it doesn't match the original password that should be all we care about here.